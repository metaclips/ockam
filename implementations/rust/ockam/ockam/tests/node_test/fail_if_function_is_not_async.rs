#[ockam::test]
fn my_test(c: &mut ockam_node::Context) -> ockam_core::Result<()> {
    c.stop().await.unwrap();
}

fn main() {}
