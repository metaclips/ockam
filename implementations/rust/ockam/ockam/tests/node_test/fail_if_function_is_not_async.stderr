error: the `async` keyword is missing from the function declaration
 --> tests/node_test/fail_if_function_is_not_async.rs:2:1
  |
2 | fn my_test(c: &mut ockam_node::Context) -> ockam_core::Result<()> {
  | ^^
